lib/ebanq_api/profile.rb

```diff
@@ -28,12 +28,18 @@ def show(uid)
     #
     # ==== Attributes
     #
-    # * +uid+ (required) - The id of the user
-    # * +fields+ (required) - hash of params which needs to update,
+    # * +uid+ (required) - The id of the user (Number)
+    # * +params+ (required) - hash of params which needs to update,
     # e.g. {first_name: 'John', last_name: 'Doe'}
-    def update(uid, fields)
+    def update(uid, params)
       @client.make_request :post,
-                           auth_path(uid), fields
+                           auth_path(uid), params
+    end
+
+    # Get list of allowed documents
+    # /api/v1/profile/
+    def list_of_allowed_documents
+      @client.make_request :post, PROFILE_API_PATH
```

Comment:
it should be :get here (it was an issue from previuos branch which is not merged)